docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -36,109 +41,118 @@ $ sudo snap install edgexfoundry --channel=2.0
 
 ## Viewing logs
 To view the logs for all services in an EdgeX snap use the `snap log` command:
 
 ```bash
-$ sudo snap logs edgexfoundry
+sudo snap logs edgexfoundry
 ```
 
 Individual service logs may be viewed by specifying the service name:
 
 ```bash
-$ sudo snap logs edgexfoundry.consul
+sudo snap logs edgexfoundry.consul
 ```
 
 Or by using the systemd unit name and `journalctl`:
 
 ```bash
-$ journalctl -u snap.edgexfoundry.consul
+journalctl -u snap.edgexfoundry.consul
 ```
 
-These techniques can be used with any snap including application and device services snaps.
+These techniques can be used with any snap including application snap and device services snaps.
 
 ## Security services
-The EdgeX snaps have security (Secret Store and API Gateway) enabled by default. As of Ireland, the security services in the
-edgexfoundry snap consitute the following components:
 
-    - kong
-    - postgres
-    - vault
-    - security-bootstrapper-redis**
-    - security-consul-bootstrapper**
-    - security-secretstore-setup**
-    - security-proxy-setup**
+Currently, The EdgeX snap has security (Secret Store and API Gateway) enabled by default. The security services constitute the following components:
+
+    - kong-daemon (API Gateway a.k.a. Reverse Proxy)
+    - postgres (kong's database)
+    - vault (Secret Store)
+
+Oneshot services which perform the necessary security setup and stop, when listed using `snap services`, they show up as `enabled/inactive`:
 
-** these services are all 'oneshot' services, when listed using `snap services`, they show up as `enabled/inactive`.
+    - security-proxy-setup (kong setup)
+    - security-secretstore-setup (vault setup)
+    - security-bootstrapper-redis (secure redis setup)
+    - security-consul-bootstrapper (secure consul setup)
 
 Vault is known within EdgeX as the Secret Store, while Kong+PostgreSQL are used to provide the EdgeX API Gateway.
 
-For more details please refer to the snap's [API Gateway](https://github.com/edgexfoundry/edgex-go/blob/main/snap/README.md#api-gateway) documentation.
+For more details please refer to the snap's [Secret Store](https://github.com/edgexfoundry/edgex-go/blob/main/snap/README.md#secret-store) and [API Gateway](https://github.com/edgexfoundry/edgex-go/blob/main/snap/README.md#api-gateway) documentation.
 
-## Device service snaps
+## Device Service Snaps
 The following is the current list EdgeX 2.x device service snaps:
 
   * [Device Camera](https://github.com/edgexfoundry/device-camera-go/blob/main/snap/README.md)
   * [Device Modbus](https://github.com/edgexfoundry/device-modbus-go/blob/main/snap/README.md)
   * [Device MQTT](https://github.com/edgexfoundry/device-mqtt-go/blob/main/snap/README.md)
   * [Device REST](https://github.com/edgexfoundry/device-rest-go/blob/main/snap/README.md)
+  * [Device SNMP](https://github.com/edgexfoundry/device-snmp-go/blob/main/snap/README.md)
```

Comment:
Please replace all with store links to be consistent with app-service-configurable above. On the store, there will be additional information on getting started.

Reply:
Replaced with store links.